### PR TITLE
[docs] typos and updates to voyager pages.

### PR DESCRIPTION
**Pull request by aishwarya24**
@netlify /preview/migrate/known-issues/#unsupported-features

### CODE REVIEW

File: docs/content/preview/migrate/reference/datatype-mapping-oracle.md [hunk 1]
```diff
@@ -45,20 +45,20 @@ The following table includes a list of supported data type mappings for migratin
...
+| BFILE | BYTEA | Not supported. |
+| ROWID | OID | Currently, import schema is not supported. Data import results in an error. |
+| UROWID [(size)] | OID | Currently, import schema is not supported. Data import results in an error. |
+| SYS.AnyData | ANYDATA | Not supported. |
+| SYS.AnyType | ANYTYPE | Not supported. |
+| SYS.AnyDataSet | ANYDATASET | Not supported. |
+| XMLType | XML | Currently, import schema is not supported. Data import results in an error. |
+| URIType | URITYPE | Not supported. |
 
 #### ANSI supported data types
```
Comment (ddhodge) [inline docs/content/preview/migrate/reference/datatype-mapping-oracle.md:63]: why h4?

**Review by ddhodge — APPROVED** [review]

File: docs/content/preview/migrate/known-issues/_index.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ Currently, yb-voyager doesn't support the following features:
 | Feature | Description/Alternatives  | GitHub Issue |
 | :------ | :------------------------ | :----------- |
 | ALTER VIEW | YugabyteDB does not yet support any schemas containing `ALTER VIEW` statements. | [48](https://github.com/yugabyte/yb-voyager/issues/48) |
-| BLOB | yb-voyager currently ignores all columns of type BLOB. <br>Use another mechanism to load the attributes.| [43](https://github.com/yugabyte/yb-voyager/issues/43) |
+| Unsupported datatypes | Data migration is unsupported for some datatypes such as BLOB and XML. For others such as ANY and BFile, both schema and data migration is unsupported. Refer to [datatype mapping](../reference/datatype-mapping-oracle/) for the detailed list. | [43](https://github.com/yugabyte/yb-voyager/issues/43) |
```
[the remark this answers is not in the document]
Comment (shubham-yb) [reply docs/content/preview/migrate/known-issues/_index.md:26]: Please remove the GH issues link (..43)

File: docs/content/preview/migrate/reference/datatype-mapping-oracle.md [hunk 3]
```diff
@@ -45,20 +45,20 @@ The following table includes a list of supported data type mappings for migratin
...
-| UROWID [(size)] | OID | Currently, import schema is not supported. Failed SQL statements can be found in `export-dir/schema/failed.sql` |
-| SYS.AnyData | ANYDATA | Currently, import schema is not supported. Failed SQL statements can be found in `export-dir/schema/failed.sql` |
-| SYS.AnyType | ANYTYPE | Currently, import schema is not supported. Failed SQL statements can be found in `export-dir/schema/failed.sql` |
-| SYS.AnyDataSet | ANYDATASET | Currently, import schema is not supported. Failed SQL statements can be found in `export-dir/schema/failed.sql` |
-| XMLType | XML | Currently, import schema is not supported. Failed SQL statements can be found in `export-dir/schema/failed.sql` |
-| URIType | URITYPE | Currently, import schema is not supported. Failed SQL statements can be found in `export-dir/schema/failed.sql` |
+| NUMBER(6,-2) | NUMERIC(6,-2) | Results in an error because negative scale is currently not supported. Refer to the [GitHub issue](https://github.com/yugabyte/yb-voyager/issues/779) for more details.
+| BLOB | BYTEA | Data is ignored during export. Currently, import data is not supported for BLOB/CLOB. Only the schema migration is allowed.  |
```
Comment (shubham-yb) [inline docs/content/preview/migrate/reference/datatype-mapping-oracle.md:51]: Can we please replace from the second line, ie. `Currently...allowed`
with
```
Please use another mechanism to load the values.
```

File: docs/content/preview/migrate/reference/datatype-mapping-oracle.md [hunk 4]
```diff
@@ -45,20 +45,20 @@ The following table includes a list of supported data type mappings for migratin
...
+| CLOB | TEXT | Data migration is allowed to a limit of 235MB per file. |
+| NCLOB | TEXT | Data migration is allowed to a limit of 235MB per file. |
+| BFILE | BYTEA | Not supported. |
+| ROWID | OID | Currently, import schema is not supported. Data import results in an error. |
+| UROWID [(size)] | OID | Currently, import schema is not supported. Data import results in an error. |
```
Comment (shubham-yb) [inline docs/content/preview/migrate/reference/datatype-mapping-oracle.md:56]: For both rowid and urowid, currently import schema **is** supported

File: docs/content/preview/migrate/reference/datatype-mapping-oracle.md [hunk 5]
```diff
@@ -45,20 +45,20 @@ The following table includes a list of supported data type mappings for migratin
...
+| SYS.AnyData | ANYDATA | Not supported. |
+| SYS.AnyType | ANYTYPE | Not supported. |
+| SYS.AnyDataSet | ANYDATASET | Not supported. |
+| XMLType | XML | Currently, import schema is not supported. Data import results in an error. |
```
Comment (shubham-yb) [inline docs/content/preview/migrate/reference/datatype-mapping-oracle.md:60]: Currently import schema **is** supported for XML.

File: docs/content/preview/migrate/reference/datatype-mapping-oracle.md (hunk 3 above)
Comment (ddhodge) [reply docs/content/preview/migrate/reference/datatype-mapping-oracle.md:51]: another mechanism - any tips or examples of what that might be?

File: docs/content/preview/migrate/known-issues/_index.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (aishwarya24) [reply docs/content/preview/migrate/known-issues/_index.md:26]: > why link to second tab (oracle)?

The Oracle page has the full list of datatypes.

File: docs/content/preview/migrate/reference/datatype-mapping-oracle.md (hunk 3 above)
Comment (shubham-yb) [reply docs/content/preview/migrate/reference/datatype-mapping-oracle.md]: It's upto the user to decide that. They may use some other tool to do the task or manually import the values. We officially cannot suggest any such methods without proper research from our end.

**Review by shubham-yb — APPROVED** [review]

**Pull request merged**